modules/puppet/manifests/puppetserver.pp

```diff
@@ -40,13 +32,6 @@
   Class['::puppet::puppetserver::generate_cert']
     ~> Class['::puppet::puppetserver::nginx']
 
-  if $sentry_dsn {
-    class { '::puppet::puppetserver::sentry':
-      require => Class['::puppet::puppetserver::package'],
-      notify  => Class['::puppet::puppetserver::service'],
-    }
-  }
```

Comment:
Unfortunately, just removing this code won't actually remove the configuration for it (https://serverfault.com/questions/455657/how-to-revert-a-configuration-with-puppet). Instead you'll need to first merge and deploy a change with `ensure => absent` and once that's run on all our machines, you can then merge this in.

The added complication here is that we wrote the `puppet::puppetserver::sentry` class and it doesn't accept an `ensure` parameter. So instead I'd suggest changing this line https://github.com/alphagov/govuk-puppet/blob/a1cd0a425e6d9c1cf9f321e91f39bf3cd87db39a/modules/puppet/manifests/puppetserver/sentry.pp#L12 to be `ensure => absent` and removing this https://github.com/alphagov/govuk-puppet/blob/a1cd0a425e6d9c1cf9f321e91f39bf3cd87db39a/modules/puppet/manifests/puppetserver/sentry.pp#L7-L9 so it doesn't run next time.

Reply:
D'oh, I'd forgotten that! 🤦 Good catch. I've raised #10920.